OpenRA.Mods.Common/Activities/Air/ResupplyAircraft.cs

```diff
@@ -48,12 +50,23 @@ public override Activity Tick(Actor self)
 				}
 				else
 				{
-					// Helicopters should take off from their helipad immediately after resupplying.
-					// HACK: Append NextActivity to TakeOff to avoid moving to the Rallypoint (if NextActivity is non-null).
-					inner = ActivityUtils.SequenceActivities(
-						aircraft.GetResupplyActivities(host)
-						.Append(new AllowYieldingReservation(self))
-						.Append(new TakeOff(self)).Append(NextActivity).ToArray());
+                    // Helicopters should take off from their helipad immediately after resupplying,
+                    // unless the "Enter" or "Return to Base" command was explictly given by user
```

Comment:
That is a feature that we went out of our way to include: planes idle on the airfield, but will take off if another plane wants to reload or a new plane is ready to be built.